modules/uhttp/http_options.go

```diff
@@ -20,17 +20,17 @@
 
 package uhttp
 
-// ModuleOption is a function that configures module creation.
-type ModuleOption func(*moduleOptions) error
+// ModuleOptionFn is a function that configures module creation.
+type ModuleOptionFn func(*moduleOption) error
```

Comment:
This should still be named ModuleOption - this is consistent with most golang libraries. That this happens to be a function is of no concern to the user.